src/mumble/MainWindow.cpp

```diff
@@ -333,7 +333,7 @@ void MainWindow::setupGui()  {
 
 	updateTransmitModeComboBox();
 
-#if QT_VERSION < 0x050000 && !defined(Q_OS_MAC)
+#if defined(Q_OS_WIN) || (QT_VERSION < 0x050000 && !defined(Q_OS_MAC))
```

Comment:
I don’t mind either way. Mine was simpler/shorter, yours is more stuff but maybe more consistent in its two cases.

I changed to your snipped (plus missing parenthesis) and will land it.